test/get.allow.token.test.ts

```diff
@@ -0,0 +1,24 @@
+import { RelayingServices } from '../src';
+import { MockRelayingServices } from './mock';
+import Expect = jest.Expect;
+
+declare const expect: Expect;
+
+describe('Tests for get allow token', () => {
+    let sdk: RelayingServices;
+
+    beforeEach(async () => {
+        sdk = new MockRelayingServices();
+        await sdk.initialize({});
+    });
+
+    it('Should run get allow token', async () => {
+        try {
+            const allowTokens = await sdk.getAllowedTokens();
+            expect(allowTokens.length).toBeGreaterThan(0);
+            expect.arrayContaining(allowTokens);
```

Comment:
this isn't correct. please see the original comment [here](https://github.com/infuy/relaying-services-sdk/pull/2#discussion_r711374079)

Reply:
Fixed with:
```typescript
expect([]).toEqual(expect.arrayContaining(allowTokens));
```